cilium/cmd/cleanup.go

```diff
@@ -56,17 +70,63 @@ const (
 
 func init() {
 	rootCmd.AddCommand(cleanupCmd)
-	cleanupCmd.Flags().BoolVarP(&force, "force", "f", false, "Skip confirmation")
+
+	cleanupCmd.Flags().BoolVarP(&cleanAll, allFlagName, "", false, "Remove all cilium state")
+	cleanupCmd.Flags().BoolVarP(&cleanBPF, bpfFlagName, "", false, "Remove BPF state")
+	cleanupCmd.Flags().BoolVarP(&force, forceFlagName, "f", false, "Skip confirmation")
+
+	bindEnv(allFlagName, cleanCiliumEnvVar)
```

Comment:
please use the helper that we have for to generate the environment variables
```
import "github.com/cilium/cilium/pkg/option"

option.BindEnv(cleanAll)
option.BindEnv(cleanBPF)
```

For backwards compatibility let's keep the old environment variables (`CLEAN_CILIUM_STATE` and `CLEAN_CILIUM_BPF_STATE`) for a couple releases.